packages/runtime/container-runtime/src/test/gc/gcStats.spec.ts

```diff
@@ -32,7 +32,8 @@ import { configProvider } from "./gcUnitTestHelpers";
 
 describe("Garbage Collection Stats", () => {
 	const defaultSnapshotCacheExpiryMs = 5 * 24 * 60 * 60 * 1000;
-	const sweepTimeoutMs = defaultSessionExpiryDurationMs + defaultSnapshotCacheExpiryMs + oneDayMs;
+	const tombstoneTimeoutMs =
```

Comment:
I would keep the sweepTimeoutMs name and add sweepGracePeriod of 0 to it here rather than where it happens today. These tests are testing sweep so waiting for tombstoneTimeoutMs is confusing.

Reply:
Alternatively, in the awaits, add + sweepGraceTimeoutMs.

Reply:
I'm moving this down to those tests as well.